CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.13)
+cmake_minimum_required(VERSION 3.13...3.22)
```

Comment:
It's just a max version that it has been tested with and policies set. I added it because there was a warning about it but it actually came from the zlib CMakeLists.txt where the minimum version was 2.4.